examples/minimq-stm32h7/src/main.rs

```diff
@@ -213,19 +202,27 @@ const APP: () = {
             }
 
             if tick && (time % 1000) == 0 {
+                if client.is_connected().unwrap() {
```

Comment:
I didn't intend to commit this debug code, should I remove it?

Reply:
Yeah, let's get rid of it.